src/rust/bitbox02-rust/src/workflow/pairing.rs

```diff
@@ -14,14 +14,17 @@
 
 use arrayvec::ArrayString;
 use binascii;
-use bitbox02;
 use core::fmt::Write;
 
-/// This function takes a slice of bytes that must be at most 60 characters when base32
-/// encoded.
-pub fn create(bytes: &[u8]) -> bool {
+pub fn extra_hash_create(bytes: &[u8]) -> bool {
```

Comment:
I know this is existing code so, for the future: sounds very weird to me that a func which seemingly from the name just manipulates some bytes, actually blocks on user interaction. If I didn't see the code, only the signature, I would've never guessed that it blocks on UI.

Reply:
The reasoning is/was:

this is workflow/pairing.rs. If all workflow functions are called `create()`, then this follows this naming pattern.

The `_create` naming pattern was carried over from C, where the workflows are/were named like this.

The workflows I ported over didn't follow this and are just named `confirm()` etc.